build/Targets/VSL.Imports.targets

```diff
@@ -270,6 +270,23 @@
     </NuGetPackageToIncludeInVsix>
   </ItemDefinitionGroup>
 
+  <!-- Currently, the Willow installer can't install vsixes that have a <Prerequisites> element
+       but the Dev15 RC SDK packages can NOT install vsixes without them as part of the build.
+       To work around this conflict, we include them in our manifest files, but if we're building
+       with something older than Dev15 RC (as we do for official builds that we insert into setup)
+       we'll run a transform to strip out the <Prerequisites> element. -->
+  <Target Name="RewriteVsixManifestOnOfficialBuild"
+          BeforeTargets="FindSourceVsixManifest"
+          Condition="'$(IsDev15RC)' != 'true'">
+    <XslTransformation XslInputPath="$(MSBuildThisFileDirectory)TransformVsixManifest.xslt"
+                       XmlInputPaths="source.extension.vsixmanifest"
+                       OutputPaths="$(IntermediateOutputPath)source.extension.vsixmanifest" />
```

Comment:
I'd feel better if this targeted some unique subfolder. I feel like I've seen the usual rewriter might write to this location which might have two things writing to the same folder.